chipy_org/templates/_rsvp.html

```diff
@@ -5,14 +5,14 @@
     <strong>Change your RSVP: </strong>
   {% else %}
   {% if curr_meeting.reg_close_date %}
-      <p style="color: red">Registration for this event will close on 
+      <p style="color: red">Registration for this event will close on
           {{ curr_meeting.reg_close_date|date:"l F j" }} at
           {{ curr_meeting.reg_close_date|date:"g:i a" }}
       </p>
   {% endif %}
   {% endif %}
+  <!-- TODO make RSVP choices read from choices -->
```

Comment:
can this get an issue created and have it put in that TODO comment so someone could take it on?